Package.swift

```diff
@@ -12,14 +10,19 @@ let package = Package(
     dependencies: [
         .package(url: "https://github.com/apple/swift-nio.git", from: "2.48.0"),
         .package(url: "https://github.com/apple/swift-nio-ssl.git", from: "2.23.0"),
-        .package(url: "https://github.com/apple/swift-collections.git",  .upToNextMajor(from: "1.0.0"))
+        .package(url: "https://github.com/apple/swift-collections.git", .upToNextMajor(from: "1.0.0")),
     ],
     targets: [
         .target(
             name: "AMQPProtocol",
             dependencies: [
                 .product(name: "NIOCore", package: "swift-nio"),
-            ]),
+            ],
+            swiftSettings: [
+                SwiftSetting.unsafeFlags(["-Xfrontend", "-strict-concurrency=complete"]),
```

Comment:
Afaik unsafe flags are not allowed in https://swiftpackageindex.com/funcmike/rabbitmq-nio - please double check it

Reply:
I removed it (only used to get sendable warnings in older compiler versions afaik)